### ISSUE TITLE
Lost ability to get translation keys since `.tr()` method was injected into the generated file for all properties

### ISSUE THREAD

**Issue by svonidze** [rinlv/easy_localization_generator#7]
Getting translation keys is important in some cases when need to implement dynamic decision logic.

So this code
```
static const languageTypeNameMap = {
    LanguageType.known: LocaleKeys.languageTypeEnumKnown,
    LanguageType.learning: LocaleKeys.languageTypeEnumLearning,
    LanguageType.used: LocaleKeys.languageTypeEnumUsed,
  };
```
will be compiled with already set up translations.

Once a user changes the app language the `languageTypeNameMap` will still keep translations of the prev language.

**Comment by svonidze** [comment]
I recovered this ability with a new option `immediateTranslationEnabled` in `@SheetLocalization`
https://github.com/rinlv/easy_localization_generator/pull/6/commits/104ed4b32bbcd7e8dc7c91bb89807d87eb9af683

probably we have to go further and generate 2 properties
```
static const foo = 'app.tab';
static String get fooTr => foo.tr();
```
so everyone will be happy.

**Comment by rinlv** [comment]
merged #6 @svonidze
